File: app/src/main/java/com/android/keeper/recycle_items/ReminderItem.java
package com.android.keeper.recycle_items;

public class ReminderItem {

    private int reminderId;
    private String reminderTitle;
    private String reminderDate;
    private String reminderTime;
    private boolean reminderDone;

    public ReminderItem(int reminderId, String reminderTitle, String reminderDate, String reminderTime, boolean reminderDone) {
        this.reminderId = reminderId;
        this.reminderTitle = reminderTitle;
        this.reminderDate = reminderDate;
        this.reminderTime = reminderTime;
        this.reminderDone = reminderDone;
    }

    public int getReminderId() {
        return reminderId;
    }

    public String getReminderTitle() {
        return reminderTitle;
    }

    public void setReminderTitle(String reminderTitle) {
        this.reminderTitle = reminderTitle;
    }

    public String getReminderDate() {
        return reminderDate;
    }

    public void setReminderDate(String reminderDate) {
        this.reminderDate = reminderDate;
    }

    public String getReminderTime() {
        return reminderTime;
    }

    public void setReminderTime(String reminderTime) {
        this.reminderTime = reminderTime;
    }

    public boolean isReminderDone() {
        return reminderDone;
    }

    public void setReminderDone(boolean reminderDone) {
        this.reminderDone = reminderDone;
    }
}

File: app/src/main/java/com/android/keeper/recycle_items/ReminderItem.kt
package com.android.keeper.recycle_items

import android.support.test.InstrumentationRegistry
import android.support.test.runner.AndroidJUnit4
import org.junit.runner.RunWith

class ReminderItem(private val reminderId: Int, private var reminderTitle: String?, private var reminderDate: String?, private var reminderTime: String?, private var reminderDone: Boolean) {
    fun getReminderId(): Int {
        return reminderId
    }

    fun getReminderTitle(): String? {
        return reminderTitle
    }

    fun setReminderTitle(reminderTitle: String?) {
        this.reminderTitle = reminderTitle
    }

    fun getReminderDate(): String? {
        return reminderDate
    }

    fun setReminderDate(reminderDate: String?) {
        this.reminderDate = reminderDate
    }

    fun getReminderTime(): String? {
        return reminderTime
    }

    fun setReminderTime(reminderTime: String?) {
        this.reminderTime = reminderTime
    }

    fun isReminderDone(): Boolean {
        return reminderDone
    }

    fun setReminderDone(reminderDone: Boolean) {
        this.reminderDone = reminderDone
    }

}